### PR TITLE
Update to #164

### PR DESCRIPTION
**Pull request by lgarceau768**
Fix to the issue detailed in [#164](https://github.com/GSA/usagov-benefits-eligibility/issues/164)

### CODE REVIEW

File: components/DateInput.vue [hunk 1]
```diff
@@ -105,6 +105,15 @@ export default {
       default: false,
     },
   },
+  watch: {
+    dateResponse() {
+      if (this.check === "") {
```
Comment (fpigeonjr) [inline components/DateInput.vue:110]: think I may have renamed `check` to `errorMessage`.

File: components/DateInput.vue [hunk 2]
```diff
@@ -105,6 +105,15 @@ export default {
...
+        this.month = this.pullDateValue(this.dateResponse, 0)
+        this.day = this.pullDateValue(this.dateResponse, 1)
+        this.year = this.pullDateValue(this.dateResponse, 2)
+      }
+    },
+  },
   data() {
```
Comment (fpigeonjr) [inline components/DateInput.vue:117]: Linter issues: 
1. The "data" property should be above the "watch" property on line 108.eslint[vue/order-in-components](https://eslint.vuejs.org/rules/order-in-components.html)
2. The "computed" property should be above the "watch" property on line 108.eslint[vue/order-in-components](https://eslint.vuejs.org/rules/order-in-components.html)

**Review by fpigeonjr — CHANGES_REQUESTED** [review]
@lgarceau768 , found a few things here to review.

File: components/DateInput.vue (hunk 2 above)
Comment (lgarceau768) [reply components/DateInput.vue]: Fixed!

**Review by fpigeonjr — APPROVED** [review]
looks good, thanks @lgarceau768

**Pull request merged**